### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bengunvl**

### CODE REVIEW

**Comment by bengunvl** [comment]
Made some adjustments with markdown.

File: README.md [hunk 1]
```diff
@@ -6,15 +6,34 @@ The promotions resource is a representation of a special promotion or sale that
...
 vagrant up
-//If running in Windows, you might get a prompt from Windows Defender to allow Ruby to run. Click Allow access.
+```
+
+If running in Windows, you might get a prompt from Windows Defender to allow Ruby to run. Click Allow access.
 
+
+```
 vagrant ssh
+```
```
Comment (zhangcshcn) [inline README.md]: One code block should be enough

File: README.md [hunk 2]
```diff
@@ -6,15 +6,34 @@ The promotions resource is a representation of a special promotion or sale that
 
 To Run, clone the repo
 
+
+```
 git clone https://github.com/DevOps2018-Promotions/promotions.git
+```
```
Comment (zhangcshcn) [inline README.md]: One code block should be enough.

**Comment by bengunvl** [comment]
Merge pull request with the changes.

**Pull request merged**